arm-documentdb/2015-04-08/swagger/documentdb.json

```diff
@@ -312,6 +312,7 @@
       "post": {
         "operationId": "DatabaseAccounts_RegenerateKey",
         "description": "Regenerates an access key for the specified Azure DocumentDB database account.",
+        "x-ms-long-running-operation": true,
```

Comment:
This is the same case as with the DatabaseAccounts_CreateOrUpdate where we have the location value in the response header